_posts/2023-03-11-release.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2023.10
+author: Priya
+date: 2023-03-11
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Revamped BigQuery Connector for Better User Experience
+* Now, users can remove and edit a BigQuery service file in the export section. Also, multiple service files can be added.
+* Enhanced warning system: If a user creates an export that will overwrite data already exported from another view, a warning message will appear on validation. The user will be asked to confirm if this is intentional before proceeding further.
```

Comment:
This doesn't look right. It is there only for Big Query in smart combine mode. Correct me if I am wrong.

Reply:
Yes @prmammoth . It should be a part of smart combine only.Will remove it .